no-littering.el

```diff
@@ -262,6 +262,9 @@ directories."
     (setq helm-adaptive-history-file       (var "helm/adaptive-history.el"))
     (setq helm-github-stars-cache-file     (var "helm/github-stars-cache.el"))
     (setq historian-save-file              (var "historian-save.el"))
+    (eval-after-load 'indium
+      `(make-directory ,(var "indium/") t))
+    (setq indium-workspace-file (var "indium/indium-workspaces.el"))
```

Comment:
Please teach `indium` to create the directory itself.

> should the file be named "workspaces.el" instead of "indium-worspaces.el"?

The former. (Assuming it contains information about multiple "workspaces". Otherwise just `workspace.el`.)